src/Committee/Feed/CommitteeMessage.php

```diff
@@ -6,6 +6,12 @@
 use AppBundle\Entity\Committee;
 use Symfony\Component\Validator\Constraints as Assert;
 
+/**
+ * @Assert\Expression(
+ *     expression="this.isPublished() != false || this.isSendNotification() != false",
+ *     message="Vous devez selectionner au moins une des deux checkbox"
```

Comment:
I think we can improve this message and at least translate "checkbox"

Reply:
~selectionner~
s**é**lectionner

You can double check with @guillaumedenhez or @marineb for the wording.

Reply:
Also :
case**s**

Reply:
yes @cacahouwete good point from @B-Galati : 
- s**é**lectionner
- une des deux case**s** à cocher